assets/scss/pages/_home.scss

```diff
@@ -311,7 +267,7 @@ $content-width: 1145px;
             }
         }
         .home-description.connected {
-            // Rules to don't be hidden by the .home-search-box::before background-image
+            // Rules to don't be hidden by the .home .search-box::before background-image
```

Comment:
Ici et ligne 250, 338, 373 : 

`// Rules to avoid being overridden by .home .search-box::before background-image`